underlay/js/bigipreport.js

```diff
@@ -2894,10 +3082,15 @@ function showVirtualServerDetails(virtualserver, loadbalancer) {
                     <td>${matchingvirtualserver.persistence.join('<br>')}</td>
                   </tr>
                   <tr><th>Source Translation</th><td>${xlate}</td></tr>
-                  <tr>
-                    <th>Other Profiles</th>
-                    <td>${matchingvirtualserver.otherprofiles.join('<br>')}</td>
-                  </tr>
+				  <tr>
```

Comment:
Patch removes "Other Profiles" and should not.

Reply:
fixed